client-mr/mr/src/main/java/org/apache/hadoop/mapreduce/task/reduce/CelebornShuffleConsumer.java

```diff
@@ -114,6 +114,16 @@ private ShuffleClientMetrics createMetrics(
     } catch (Exception e) {
       // ignore this exception because the createMetrics might use hadoop2
     }
+
+    // for hadoop 3.1
```

Comment:
can you mention the JIRA ticket in comments too? there are a lot of vender customized hadoop that may not align with apache hadoop.

Reply:
Attached the MR jira ticket